packages/rocketchat-ui/client/lib/fileUpload.coffee

```diff
@@ -9,6 +9,8 @@ getUploadPreview = (file, callback) ->
 	# If greater then 10MB don't try and show a preview
 	if file.file.size > 10 * 1000000
 		callback(file, null)
+	else if file.file.type is undefined
```

Comment:
maybe testing any `falsy` value would be safier.. what do you think? like `else if not file.file.type?`